metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, ctx: PipelineContext, config: DatahubRestSinkConfig):
             retry_max_times=self.config.retry_max_times,
             extra_headers=self.config.extra_headers,
             ca_certificate_path=self.config.ca_certificate_path,
+            disable_ssl_verification=config.disable_ssl_verification,
```

Comment:
nit: maybe use `self.config.` to align with previous lines?

Reply:
fixed